include/text.h

```diff
@@ -0,0 +1,4592 @@
+#define TEXTID_005F 0x5F
+#define TEXTID_0060 0x60
+#define TEXTID_0061 0x61
+#define TEXTID_0062 0x62
+#define TEXTID_0063 0x63
+#define TEXTID_0064 0x64
+#define TEXTID_0065 0x65
+#define TEXTID_0066 0x66
+#define TEXTID_0067 0x67
+#define TEXTID_0068 0x68
+#define TEXTID_0069 0x69
+#define TEXTID_006A 0x6A
+#define TEXTID_006B 0x6B
+#define TEXTID_006E 0x6E
+#define TEXTID_006F 0x6F
+#define TEXTID_0070 0x70
+#define TEXTID_0071 0x71
+#define TEXTID_0072 0x72
+#define TEXTID_0073 0x73
+#define TEXTID_0074 0x74
+#define TEXTID_0075 0x75
+#define TEXTID_0076 0x76
+#define TEXTID_0077 0x77
+#define TEXTID_0078 0x78
+#define TEXTID_0079 0x79
+#define TEXTID_007A 0x7A
+#define TEXTID_007B 0x7B
+#define TEXTID_007C 0x7C
+#define TEXTID_007D 0x7D
+#define TEXTID_007E 0x7E
+#define TEXTID_007F 0x7F
+#define TEXTID_0080 0x80
+#define TEXTID_0081 0x81
+#define TEXTID_0082 0x82
+#define TEXTID_0083 0x83
+#define TEXTID_0084 0x84
+#define TEXTID_0085 0x85
+#define TEXTID_0086 0x86
+#define TEXTID_0087 0x87
+#define TEXTID_0088 0x88
+#define TEXTID_0089 0x89
+#define TEXTID_008A 0x8A
+#define TEXTID_008B 0x8B
+#define TEXTID_008C 0x8C
+#define TEXTID_008D 0x8D
+#define TEXTID_008E 0x8E
+#define TEXTID_008F 0x8F
+#define TEXTID_0091 0x91
+#define TEXTID_0092 0x92
+#define TEXTID_0093 0x93
+#define TEXTID_0094 0x94
+#define TEXTID_0095 0x95
+#define TEXTID_0096 0x96
+#define TEXTID_0097 0x97
+#define TEXTID_0098 0x98
+#define TEXTID_0099 0x99
+#define TEXTID_009A 0x9A
+#define TEXTID_009B 0x9B
+#define TEXTID_009C 0x9C
+#define TEXTID_009D 0x9D
+#define TEXTID_009E 0x9E
+#define TEXTID_009F 0x9F
+#define TEXTID_00A0 0xA0
+#define TEXTID_00A1 0xA1
+#define TEXTID_00A2 0xA2
+#define TEXTID_00A3 0xA3
+#define TEXTID_00A4 0xA4
+#define TEXTID_00A9 0xA9
+#define TEXTID_00AA 0xAA
+#define TEXTID_00AB 0xAB
+#define TEXTID_00AC 0xAC
+#define TEXTID_00AD 0xAD
+#define TEXTID_00AE 0xAE
+#define TEXTID_00AF 0xAF
+#define TEXTID_00B0 0xB0
+#define TEXTID_00B1 0xB1
+#define TEXTID_00B2 0xB2
+#define TEXTID_00B4 0xB4
+#define TEXTID_00B5 0xB5
+#define TEXTID_00B6 0xB6
+#define TEXTID_00B7 0xB7
+#define TEXTID_00B8 0xB8
+#define TEXTID_00B9 0xB9
+#define TEXTID_00BA 0xBA
+#define TEXTID_00BB 0xBB
+#define TEXTID_00BC 0xBC
+#define TEXTID_00BD 0xBD
+#define TEXTID_00BE 0xBE
+#define TEXTID_00BF 0xBF
+#define TEXTID_00C0 0xC0
+#define TEXTID_00C4 0xC4
+#define TEXTID_00C5 0xC5
+#define TEXTID_00C6 0xC6
+#define TEXTID_00C7 0xC7
+#define TEXTID_00C8 0xC8
+#define TEXTID_00C9 0xC9
+#define TEXTID_00CA 0xCA
+#define TEXTID_00CB 0xCB
+#define TEXTID_00CC 0xCC
+#define TEXTID_00CD 0xCD
+#define TEXTID_00F2 0xF2
+#define TEXTID_00F3 0xF3
+#define TEXTID_00F4 0xF4
+#define TEXTID_00F5 0xF5
+#define TEXTID_00F6 0xF6
+#define TEXTID_00F7 0xF7
+#define TEXTID_00F8 0xF8
+#define TEXTID_00F9 0xF9
+#define TEXTID_00FA 0xFA
+#define TEXTID_00FB 0xFB
+#define TEXTID_00FC 0xFC
+#define TEXTID_00FD 0xFD
+#define TEXTID_00FE 0xFE
+#define TEXTID_OPEN_THE_SELECT_ITEM_SCREEN 0xFF
```

Comment:
```suggestion
#define TEXTID_OPEN_SELECT_ITEM_SCREEN 0xFF
```
Don't think the `THE` is necessary, and would prefer it be as short as possible.